typings/index.d.ts

```diff
@@ -0,0 +1,31 @@
+import { ComponentType } from 'react';
+
+type StrengthFunction = (
+    rect: {
+        x: number; 
+        y: number;
+        w: number;
+        h: number;
+    }, 
+    point: {
+        x: number;
+        y: number;
+    }
+) => number;
+
+declare function createHorizontalStrength(_buffer: number): StrengthFunction;
+declare function createVerticalStrength(_buffer: number): StrengthFunction;
+
+declare const defaultHorizontalStrength: StrengthFunction;
+declare const defaultVerticalStrength: StrengthFunction;
+
```

Comment:
Exports like `createVerticalStrength` are missing here and causing import errors. Would be nice to fix that too

`@spotware/react-dnd-scrollzone/typings/index"' has no exported member 'createVerticalStrength'.`